ui/src/app/utils/resourceActions.ts

```diff
@@ -41,4 +54,40 @@ export const cohortActionList = [
 
 export const resourceActionList =  notebookActionList.concat(cohortActionList);
 
+export function convertToResources(list: any[], component): RecentResource[] {
+  const resourceList = [];
+  if (component.resourceType === ResourceType.NOTEBOOK) {
```

Comment:
Do you access anything from component other than `resourceType`? Can we just pass in resource type?

Reply:
yup this.workspaceName, this.workspace, etc, in convertToResource...